google_web_risk/help.md

```diff
@@ -70,6 +70,7 @@ Google's Web Risk API requires an API key for use. Sign up for the beta [here](h
 
 # Version History
 
+* 2.0.2 - Update help.md
```

Comment:
We can remove this line since the we didn't release any plugins with the last version in changelog as `New spec and help.md format for the Hub`.

Reply:
Will do

Reply:
Resolved by latest commit.